server.js

```diff
@@ -1,34 +1,34 @@
 // TASK 1: Setting up server
 
 
-const http = require('http');
+// const http = require('http');
 
-const server = http.createServer((req, res) => {
-  res.statusCode = 200;
-  res.setHeader('Content-Type', 'text/plain');
-  res.end('Hello World\n');
-});
 
-const PORT = 8000;
-const HOSTNAME = 'localhost';
 
-server.listen(PORT, HOSTNAME, () => {

```

Comment:
The 'mongoUri' variable seems to be unrelated to the rest of the code. It should be removed or placed in an appropriate context.